OpenLisp.NET.sln

```diff
@@ -17,10 +17,6 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "OpenLisp.Terminal", "OpenLi
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "OpenLisp.Repl", "OpenLisp.Repl\OpenLisp.Repl.csproj", "{648A6B9E-B246-4E70-93C8-4483D0B91435}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "OpenLisp.Machine.Kernel", "OpenLisp.Machine.Kernel\OpenLisp.Machine.Kernel.csproj", "{96A59125-9AE5-4E91-8741-B5214B469450}"
-EndProject
-Project("{471EC4BB-E47E-4229-A789-D1F5F83B52D4}") = "OpenLisp.Machine.KernelBoot", "OpenLisp.Machine.Kernel\OpenLisp.Machine.KernelBoot.Cosmos", "{31DC0EAE-9D92-42CB-9796-5B8CFB08486A}"
-EndProject
```

Comment:
The `COSMOS` related projects should probably go in their own domain-specific solution.  I imagine the solution file can stay in this same repo.